test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -13452,6 +13453,21 @@ def create_vnode_def(self, name, attrs={}, numnodes=1, sharednode=True,
         del attribs
         return "\n".join(vdef)
 
+    def add_checkpoint_script(self, dirname=None, body=None, mode=None,
```

Comment:
There are more than one checkpoint $action scripts in the mom config.  Call this function add_checkpoint_abort_script()